edx/edx-enterprise
# Add missing migration file for string updates

**Description:** @zubair-arbi @grantgoodmanedX @haikuginger here is the missing migration file for the string updates made for ENT-171 / https://github.com/edx/edx-enterprise/pull/75

**JIRA:** ENT-171

**Dependencies:** None
**Merge deadline:** ASAP

**Installation instructions:** Check out branch!

**Testing instructions:**

1. Set up venv using branch
2. Run `./manage.py migrate`
3. Should be good!

**Merge checklist:**


**Post merge:**

**Author concerns:** None
